scripts/apps/authoring/metadata/views/metadata-list-item.html

```diff
@@ -0,0 +1,6 @@
+<li class="basic-list__item">
+    <label class="basic-list__item-label">{{label}}</label>
+    <div class="basic-list__item-data">
+        {{value}}
```

Comment:
imo it would make sense to use transclude here so you could also use it for things like:
```
<li sd-metadata-list-item data-label="Updated">
    <sd-reldate-complex datetime="item.versioncreated"></sd-reldate-complex>
</li>
```